src/common/options/ceph-exporter.yaml.in

```diff
@@ -0,0 +1,39 @@
+# -*- mode: YAML -*-
+---
+
+options:
+- name: sock_dir
+  type: str
+  level: advanced
+  desc: The path to ceph daemons socket files dir
+  default: /var/run/ceph/
+  services:
+  - ceph-exporter
+- name: exporter_addrs
+  type: str
+  level: advanced
+  desc: Host ip address where exporter is deployed
+  default: 127.0.0.1
+  services:
+  - ceph-exporter
+- name: exporter_port
```

Comment:
Just for consistency, in other daemons (e.g.: mon) the port is defined together with the ip in the `public_addr` option (`ip:port`). In fact that setting is defined for most daemons (global.yml.in).